del_events.py

```diff
@@ -44,28 +47,35 @@ def get_credentials():
         flow.user_agent = APPLICATION_NAME
         if flags:
             credentials = tools.run_flow(flow, store, flags)
-        else: # Needed only for compatibility with Python 2.6
+        else:  # Needed only for compatibility with Python 2.6
```

Comment:
Okay, I might have been mistaken in my previous comment. Is this some kind of python coding convention?

Reply:
Yes, it's a convention for comments.